src/com/serotonin/mango/rt/dataImage/exceptions_level/ExceptionsArea.java

```diff
@@ -0,0 +1,26 @@
+package com.serotonin.mango.rt.dataImage.exceptions_level;
+
+import com.serotonin.mango.rt.dataImage.PointValueTime;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.List;
+
+public class ExceptionsArea {
+
+    public Log LOG = LogFactory.getLog(ExceptionsArea.class);
+
+    public boolean doListIsNull(List<PointValueTime> pointValueTimes){
+
+        if(pointValueTimes == null) {
+            try {
+                throw  new Exception("We have problem with get needed data From database.");
+            } catch (Exception e) {
+                LOG.error(e.getMessage());
+            }
```

Comment:
I see no sense in such a construction. You use the exception to interrupt some methods, mark the place in the code that can cause the problem, here you return the exception and you catch it to display the message in the logs ... You should use the logger here. Not to mention that the exception should not be an Exception, maybe an IllegalStateException, IllegalArgumentException? If there is no proper exception in standard libraries, then we create a new type, we rather avoid simply using Exception.